docs/end-to-end-demo.md

```diff
@@ -97,7 +97,10 @@ carry out one or the other (or adapt to your distribution), then continue on wit
     alias kubectl='microk8s kubectl'
     alias helm='microk8s helm3'
     ```
-1. Enable privileged pods and restart microk8s.
+1. For the sake of this demo, the udev video broker pods run privileged to easily grant them access to video devices, so
```

Comment:
When you set the rw permission as part of DeviceSpec, I have noticed that Kubelet adds those to others as well, so the privileged should not be needed if you could pass the permissions as part of the Configuratoin.

Reply:
Great. It seems like this should be left as is for now and then once the Agent starts using DeviceSpec instead of Mounts and permissions are made settable via the Configuration, we can remove this?

Reply:
Right, makes sense.